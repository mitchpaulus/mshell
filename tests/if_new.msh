true  ("true branch") ("false branch") iff wl
false ("true branch") ("false branch") iff wl

false ("true branch")   (
false ("first else if") (
false ("second else if")
      ("final else") iff) iff) iff wl

# Test true only branch logic
true  ("Got true\n" w) iff
false ("Should not get true\n" w) iff
0 ("Got true\n" w) iff
123 ("Should not get true\n" w) iff
