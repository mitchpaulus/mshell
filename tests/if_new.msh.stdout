true branch
false branch
final else
Got true
Got true
